group-membership/README.md

```diff
@@ -0,0 +1,116 @@
+description: Share documents, events, and communications with users outside 
+your domain
+labels:  Sheets, Groups for Business, Apps Script
+material_icon: group_work
+create_time: 2019-08-30
+update_time: 2019-12-27
+---
+Contributed by Tech and Eco, follow me on
+[Twitter](https://twitter.com/TechandEco)!
+
+Use a Google Group to work cross-functionally with vendors, partners,
+customers, and volunteers outside of your domain, and scale access
+to work assignments and leverage security controls. You can add an external
+user's Google email (this can be their own G Suite account, a G Suite user
+account you created for them within your domain, or a free Gmail account) to
+your G Suite Google group (not available for consumer Google Groups) via a
+Google Sheet to manage users in bulk and keep track of the onboarding history.
+
+Top 6 benefits of this solution:
+
+ 1. Adds a new external user to a G Suite Google Group you create
+   (example: externalteam@mydomain.com) once, and it allows users to open all
+   resources that are shared with that group moving forward (calendar, files,
+   training site, dashboards (built on DataStudio), etc.
+
+ 1. When adding one or more users to your Google Group, users can receive a
+    welcome email with the links they can access. Creating a Google Site is
+    optional but recommended to centralize information in a
+    beautiful website interface.
+
+ 1. The template of the welcome email is composed in a Google Doc. You
+    can use seperate Google Docs for different recipients to customize
+    messaging.
+
+ 1. Capture the email addresses of users in your Sheet, and add them in bulk to
+    the desired Google Groups at a later time by changing the column "Allowed"
+    to "yes" for each row of users.
+
+ 1. You or members of your team, can add users to different Google Groups as
+    long as the person who made a copy of this sheet has
+    _manager_ or _owner_ rights in each of those Google Groups.
+
+ 1. Removing a user from a Group (via the Google Group's interface) will revoke
+    their access to all resources. You can also limit documents to be available
+    to that group for a
+    [specific period of time](https://support.google.com/docs/answer/2494893?co=GENIE.Platform%3DDesktop&hl=en).
+
+![Users are added to a Google Group via a Google Sheet](https://cdn.jsdelivr.net/gh/gsuitedevs/solutions@master/group-membership/demo.gif)
+
+## Technology highlights
+
+- Install a trigger with a click so the script is setup to run everytime the
+Sheet is edited. To learn more visit
+ [this page](https://developers.google.com/apps-script/guides/triggers/installable))
+- Add members to a _Google Group for Business_ using the
+ [Admin Directory API](https://developers.google.com/apps-script/advanced/admin-sdk-directory)
+
+## Try it
+
+1. Make a [copy of this Google Spreadsheet](https://docs.google.com/spreadsheets/d/1KJKc2DcCr2bHLCq5Judvvwuen3k3ifFY8wtQWKfHDXU/copy)     from your _G Suite_ account.
```

Comment:
Can we shorten the title of this? Sheet is already implied and via onEdit is a technical detail, how about "External Sustainability Group Members"

Reply:
love it!

Reply:
Changed, thank you!